.credo.exs

```diff
@@ -41,7 +41,7 @@
         {Credo.Check.Refactor.MatchInCondition},
         {Credo.Check.Refactor.NegatedConditionsInUnless},
         {Credo.Check.Refactor.NegatedConditionsWithElse},
-        {Credo.Check.Refactor.Nesting},
+        {Credo.Check.Refactor.Nesting, false},
```

Comment:
FYI, I'm advocating for disabling this rule. While I agree with the principle of avoiding excessive nesting _when_ it's detrimental to code clarity, I think it should be a judgement call and not an arbitrary depth number.

Reply:
Yeah agreed that it's contrived to make it a hard and fast rule